packages/mobile/src/fiatExchanges/utils.tsx

```diff
@@ -1,39 +1,59 @@
 import { getRegionCodeFromCountryCode } from '@celo/utils/lib/phoneNumbers'
```

Comment:
This `utils` file is getting pretty big, I'd be a fan of dividing it between maybe `moonpayUtils`, `simplexUtils` and maybe leave some things here. No need to do that in this PR I guess, just a thought

Reply:
Yeah you're right. I'll focus on breaking this up when we work on the redesign

Reply:
Yeah you're right. I'll plan on break this up into more modular files during the redesign work